File: client_packages_dev/js_packages/src/Authorization/AuthorizationEvents.ts
export enum AuthorizationEvents {
  LOGIN_SUCCESS_FROM_SERVER = "SERVER:CLIENT:Authorization:LoginSuccess",
  REGISTER_SUCCESS_FROM_SERVER = "SERVER:CLIENT:Authorization:RegisterSuccess",
  FIRST_CONNECTION_FROM_SERVER = "SERVER:CLIENT:Authorization:FirstConnection",
  NEED_LOGIN_FROM_SERVER = "SERVER:CLIENT:Authorization:NeedLogin",
  CHARACTER_SPAWNED_FROM_SERVER = "SERVER:CLIENT:CharacterManager:OnCharacterSpawned",

  GO_TO_CHARACTER_MANAGER = "CEF:CEF:Authorization:GoToCharacterManager",

  PLAYER_READY_TO_SERVER = "CLIENT:SERVER:Authorization:PlayerReady"
}

File: dotnet/resources/Authorization/ChainsOfResponsibility/LoginHandler/AuthorizationEvents.cs
namespace Authorization.ChainsOfResponsibility.LoginHandler
{
    internal static class AuthorizationEvents
    {
        public const string LoginFailureToCef = "SERVER:CEF:Authorization:LoginFailure";
        
        public const string LoginSuccessToClient = "SERVER:CLIENT:Authorization:LoginSuccess";
    }
}